spec/index.bs

```diff
@@ -6767,19 +6787,40 @@ interface GPUCanvasContext {
         given device.
 
         <div algorithm="GPUCanvasContext.getSwapChainPreferredFormat">
-            **Called on:** {{GPUCanvasContext}} this.
+            **Called on:** {{GPUCanvasContext}} |this|.
 
             **Arguments:**
-            <pre class=argumentdef for="GPUCanvasContext/getSwapChainPreferredFormat(device)">
-                device: Device the swap chain format should be queried for.
+            <pre class=argumentdef for="GPUCanvasContext/getSwapChainPreferredFormat(adapter)">
+                |adapter|: Adapter the swap chain format should be queried for.
             </pre>
 
-            **Returns:** {{Promise}}&lt;{{GPUTextureFormat}}&gt;
+            **Returns:** {{GPUTextureFormat}}
 
-            Issue: Describe {{GPUCanvasContext/getSwapChainPreferredFormat()}} algorithm steps.
+            <div class=content-timeline>
+                1. Let |preferredFormat| be the [$preferred swap chain format$] for |this| and |adapter|.
+                1. Return |preferredFormat|.
+            </div>
         </div>
 </dl>
 
+The <dfn dfn>supported swap chain formats</dfn> are a [=set=] of {{GPUTextureFormat}}s that must be
+supported when specified as a {{GPUSwapChainDescriptor}}.{{GPUSwapChainDescriptor/format}} regardless
+of the given {{GPUSwapChainDescriptor}}.{{GPUSwapChainDescriptor/device}}, initially set to:
+&laquo;{{GPUTextureFormat/"bgra8unorm"}}, {{GPUTextureFormat/"bgra8unorm-srgb"}},
+{{GPUTextureFormat/"rgba8unorm"}}, {{GPUTextureFormat/"rgba8unorm-srgb"}}&raquo;.
```

Comment:
@jdashg can you help us understand whether it makes sense to use srgb render target formats for rendering into normal canvases (that don't have CanvasColorEncodingEnum "unorm8-srgb")?